core/string/translation_server.cpp

```diff
@@ -503,7 +503,7 @@ String TranslationServer::get_tool_locale() {
 		// Look for best matching loaded translation.
 		Ref<Translation> t = main_domain->get_translation_object(locale);
 		if (t.is_null()) {
-			return "en";
+			return locale;
```

Comment:
This is not how it supposed to work, if no translation is provided it should return `en`, since everything assumes default locale is English. Otherwise, any untranslated app will use whatever current user locale, which will result in multiple undesired effects (flipping UI if user locale is right-to-left for example).

Reply:
If so, then I can only set the Bidi Language for each control when I don't have translation loaded? (even if I've already specified the language for the project)
Is it possible to let TextServer use `get_locale()` instead to get the current language?

Reply:
> even if I've already specified the language for the project

Not sure if there's option to set language for the project (only `internationalization/locale/test`, which should probably be used instead of hardcoded `en` if set).